src/docs/Components/DataTable/index.mdx

```diff
@@ -94,6 +150,14 @@ Each cells are available in 3 sizes.
 | Medium (default) | 3mu (48px)   |
 | Large            | 3.5mu (56px) |
 
+## Layout
+
+In order to avoid double scrolling, we recommend 2 ways of defining the height of the datatable: either by the size of the viewport, or by the number of rows displayed.
```

Comment:
@jeanBptst  I think examples are necessary to help users understand what you mean.